dist/BackgroundImage.js

```diff
@@ -80,6 +80,8 @@ var BackgroundLoader = (function (_React$Component) {
   }, {
     key: 'componentWillUnmount',
     value: function componentWillUnmount() {
+      this.image.onerror = null;
+      this.image.onload = null;
```

Comment:
Note: these two lines were not intentional changes but looks like previous dist file did not have them.